docs/books/user-guide/configuration-security.adoc

```diff
@@ -417,290 +417,367 @@ For more information about these attributes, see xref:adding-sasl-authentication
+{RouterName} uses policies to determine whether to permit a connection, and if it is permitted, to apply the appropriate resource limits.
 
 When a client creates a connection to the router, the router first determines whether to allow or deny the connection. This decision is based on the following criteria:
 
-* Whether the connection will exceed the router's global connection limit (defined in the global policy)
-* Whether the connection will exceed the vhost's connection limits (defined in the vhost policy that matches the host to which the connection is directed)
+* Whether the connection will exceed the router’s global connection limit (defined in the global policy)
 
-If the connection is allowed, the router assigns the user (the authenticated user name from the connection) to a user group, and enforces the user group's resource limits for the lifetime of the connection.
+* Whether the connection will exceed the vhost’s connection limits (defined in the vhost policy that matches the host to which the connection is directed)
 
-=== Configuring Global Policies
+If the connection is allowed, the router assigns the user (the authenticated user name from the connection) to a user group, and enforces the user group’s resource limits for the lifetime of the connection.
 
-You can set the incoming connection limit for the router and define how it should use vhost policies by configuring a global policy.
+=== Setting Global Connection Limits
+
+You can set the incoming connection limit for the router. This limit defines the total number of concurrent client connections that can be open for this router.
 
 .Procedure
 
-* In the router configuration file, add a `policy` section.
+* In the router configuration file, add a `policy` section and set the `maxConnections`.
 +
 --
 [options="nowrap",subs="+quotes"]
 ----
-policy = {
-    maxConnections: 10000  // <1>
-    enableVhostPolicy: true  // <2>
-    policyDir: /etc/qpid-dispatch/policies/  // <3>
-    defaultVhost: $default  // <4>
+policy {
+    maxConnections: 10000
 }
 ----
-<1> The maximum number of concurrent client connections allowed for this router. This limit is always enforced, even if no other policy settings have been defined. The limit is applied to all incoming connections regardless of remote host, authenticated user, or targeted vhost. The default (and the maximum) value is `65535`.
+`maxConnections`::
+This limit is always enforced, even if no other policy settings have been defined. The limit is applied to all incoming connections regardless of remote host, authenticated user, or targeted vhost. The default (and the maximum) value is `65535`.
+--
+
+=== Setting Connection and Resource Limits for Messaging Endpoints
+
+You can define the connection limit and AMQP resource limits for a messaging endpoint by configuring a _vhost policy_. Vhost policies define what clients can access on a messaging endpoint over a particular connection. 
```

Comment:
The second sentence would be clearer as "Vhost policies define what resources clients are permitted to access on a messaging endpoint over a particular connection."